internal/quickstart/toggle_flag.go

```diff
@@ -52,11 +61,19 @@ func (m toggleFlagModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case key.Matches(msg, pressableKeys.Quit):
 			return m, tea.Quit
 		case key.Matches(msg, pressableKeys.Tab):
+			if !m.flagWasFetched {
+				return m, nil
+			}
 			m.flagWasEnabled = true
 			m.enabled = !m.enabled
 			m.err = nil
 			return m, toggleFlag(m.client, m.accessToken, m.baseUri, m.flagKey, m.enabled)
 		}
+	case fetchedFlagStatusMsg:
+		m.enabled = msg.enabled
+		m.flagWasFetched = true
```

Comment:
Could we do something like this so we show "Check your blah to see the change!" if the fetched flag is already enabled?
```
if msg.enabled {
	m.flagWasEnabled = true
}
```

Reply:
I'm confused. The way the code is currently written it'll show "Check your blah to see the change!" when the flag is toggled (in general). 
I think that would make sense since the user can go back a page, maybe to look at the code snippets and figure out why their code wasn't working. So when they come back to the toggle flag page they would probably expect not to see a success message until they toggle the flag again.

Reply:
Yeah, that's fair